src/main/scala/edu/ie3/simona/model/participant2/PrimaryDataParticipantModel.scala

```diff
@@ -62,22 +61,27 @@ final case class PrimaryDataParticipantModel[PD <: PrimaryData: ClassTag](
       PrimaryDataState[PD],
     ] {
 
-  override val initialState: ModelInput => PrimaryDataState[PD] = { input =>
-    val primaryData = getPrimaryData(input.receivedData)
-    PrimaryDataState(
-      primaryData,
-      input.currentTick,
-    )
+  override val initialState: (Long, ZonedDateTime) => PrimaryDataState[PD] = {
+    (tick, _) =>
+      PrimaryDataState(
+        primaryDataExtra.zero,
```

Comment:
Why did you change the primary data?

Reply:
Because now, a state is determined in two steps (for some complicated reasons) and for the initial state, we don't supply input data anymore.